compiler-rt/lib/scudo/standalone/vector.h

```diff
@@ -116,18 +116,20 @@ template <typename T> class VectorNoCtor {
   uptr CapacityBytes = 0;
   uptr Size = 0;
 
-  T LocalData[256 / sizeof(T)] = {};
+  T LocalData[StaticNumEntries] = {};
   MemMapT ExternalBuffer;
 };
 
-template <typename T> class Vector : public VectorNoCtor<T> {
+template <typename T, size_t StaticNumEntries>
+class Vector : public VectorNoCtor<T, StaticNumEntries> {
 public:
-  constexpr Vector() { VectorNoCtor<T>::init(); }
+  static_assert(StaticNumEntries > 0U, "");
```

Comment:
How about put some words about the zero size is not allowed?

Reply:
IIt's currently in the commit message. Do you want me to make a comment directly in the code?